super-turbo-ignore.js

```diff
@@ -0,0 +1,185 @@
+// Derived from vercel/turborepo:packages/turbo-ignore/src/ignore.ts@29aeaf0
```

Comment:
Is TS supported by actions? If so, any reason for using JS over TS?

Reply:
Was for sake of simplicity since running TypeScript directly would mean going through `npx ts-node` when node is already available. I suppose a possible alternative is using [Deno](https://deno.com) or [Bun](https://bun.sh) which support running TypeScript directly. Otherwise, would need to configure this project to run `tsc` and release the artifact.

Reply:
Switching to bun/bunx since it appears to be faster (around 6 seconds to 4 seconds), not counting installation time.